src/index.js

```diff
@@ -377,6 +378,29 @@ class ServerlessAppsyncPlugin {
     return openIdConnectConfig;
   }
 
+  getLambdaAuthorizerConfig(provider) {
+    if (
+      !provider.lambdaAuthorizerConfig ||
+      !(provider.lambdaAuthorizerConfig.functionArn || provider.lambdaAuthorizerConfig.functionName)
+    ) {
+      throw new this.serverless.classes.Error('lambdaAuthorizerConfig `functionArn` or `functionName` must be present');
+    }
+
+    const lambdaAuthorizerConfig = {
+      IdentityValidationExpression: provider.lambdaAuthorizerConfig.identityValidationExpression,
+      AuthorizerResultTtlInSeconds: provider.lambdaAuthorizerConfig.authorizerResultTtlInSeconds,
+    };
+
+    if (provider.lambdaAuthorizerConfig.functionArn) {
+      lambdaAuthorizerConfig.AuthorizerUri = provider.lambdaAuthorizerConfig.functionArn;
+    } else {
+      const [functionName, functionAlias] = provider.lambdaAuthorizerConfig.functionName.split(':');
```

Comment:
I definitely like the idea of the `name:alias` notation, however, can we stick to `functionAlias` just like in the [resolver config](https://github.com/sid88in/serverless-appsync-plugin/blob/1cb865f9257edb13be86cfc771067a9facd17b73/src/index.js#L120)? For consistency. 

You probably can reuse `getLambdaArn` around here too.

We can add support for the `:` notation later in a separate PR 🙌

Reply:
this support will be great! 😃